examples/simpleapp/pom.xml

```diff
@@ -17,6 +17,27 @@
 			<groupId>io.horizen</groupId>
 			<artifactId>sidechains-sdk</artifactId>
 			<version>0.2.7</version>
+
+			<!-- exclude the default simple-sl4j library and include the bridge to log4j -->
+			<exclusions>
```

Comment:
Instead of excluding the dependency of `sidechains-sdk` and replacing it with another one, we may directly do it in core sdk pom.
So, other developers will have it correct of out of the box.

Reply:
done